### ISSUE TITLE
Remove permissions handling and put in control of user

### ISSUE THREAD

**Issue by marksyzm** [1fabiopereira/react-native-pitch-detector#43]
https://github.com/1fabiopereira/react-native-pitch-detector/blob/127fb45dac647e7fe018f8be66b90e538141da71/src/internal/permissions/index.ts#L7

This stuff can all be handled externally and bloats the scope of the repo - are you happy for me to move this out?

**Comment by 1fabiopereira** [comment]
@marksyzm No problem, you can remove them, just remember to update the doc if necessary, when you send the PR for `develop` I will review it.

**Comment by qubithaze** [comment]
Is there still any plan to do this?  I'd like to use something other than the old version of react-native-permissions that is a dependency here.

**Comment by 1fabiopereira** [comment]
@qubithaze Yes, I'm working on remove this, but it's take one or two weeks to be done.

**Comment by qubithaze** [comment]
> @qubithaze Yes, I'm working on remove this, but it's take one or two weeks to be done.

Thanks.  For the time being I’m just creating my own native module.  Let me know if you need any help!  Appreciate you open sourcing this.
